Cargo.toml

```diff
@@ -79,8 +79,8 @@ nu-test-support = { version = "0.83.1"}
 strum = "0.24.1"
 strum_macros = "0.24.3"
 
-[features]
 
+[features]
 # Enable to statically link OpenSSL; otherwise the system version will be used. Not enabled by default because it takes a while to build
 static-link-openssl = ["dep:openssl"]
```

Comment:
If we don't need it any longer, why do we have it here?

Reply:
We need it on Unix systems (not macos though). Nushell itself uses a crate which builds openssl, so on Linux we statically link at CI time.